### DISCUSSION TITLE
Bedrock Anthropic3 Exponential Backoff

### DISCUSSION THREAD

**Discussion by jledesma84** [spring-projects/spring-ai#748] [Q&A] [answered]
Hello All,

Does the Bedrock Anthropic3 client have any way to configure an exponential backoff retry policy, similar to the AzureOpenAiChatClient or OpenAiChatClient?

**Comment by maxjiang153** [comment]
Hi, could you provide some scenarios or contexts for this feature?

**Reply by maxjiang153** [reply, marked as answer]
I created a PR for this feature: https://github.com/spring-projects/spring-ai/pull/759

**Reply by jledesma84** [reply]
Awesome! That was fast!

**Reply by jledesma84** [reply]
@maxjiang153 do you think this feature can be available for version 1.0.0?
